promql/ast.go

```diff
@@ -319,3 +337,82 @@ func Children(node Node) []Node {
 		panic(errors.Errorf("promql.Children: unhandled node type %T", node))
 	}
 }
+
+// PositionRange describes a position in the input string of the parser.
+type PositionRange struct {
+	Start Pos
+	End   Pos
+}
+
+// mergeRanges is a helper function the PostionRanges of two Nodes
```

Comment:
This first comment line doesn't parse in my head (and needs a period)? Should it read:

```suggestion
// mergeRanges is a helper function to merge the PositionRanges of two Nodes.
```